README.md

```diff
@@ -1,104 +1,210 @@
 
-`qhub` is installed as a command line application in Python. It requires you to choose your the cloud provider you desire. once you've decided on a provider `qhub` will walk you through the following steps to configure your deployment:
+## QHub Cloud
+The cloud version of QHub is built using [Terraform](https://www.terraform.io/), [Helm](https://helm.sh/), and 
+[GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions).
 
-1. initialize
-2. render
-3. deploy
+### Cloud Providers
+QHub offers out-of-the-box support for [Digital Ocean], Amazon [AWS] and [GCP] (Google Cloud Provider). Although by 
+default, as long as the chosen provider offers Kubernetes services and has a Terraform module, it is possible to 
+configure QHub to deploy your project.
 
-these steps realized using the `qhub` CLI.
 
-### initialize configurations
+![image](docs/images/brand-diagram.png "architecture diagram")
 
-        qhub init do
-        qhub init aws
-        qhub init gcp
+For more details, check out the release [blog post](https://www.quansight.com/post/announcing-qhub).
 
-The `qhub init` command will generate configuration files for that service. The configutation files can be tailored to the needs of your organization. Each file specifies general project information, security, infrastructure settings, computational resource profiles and data science environments. See documentation on modifying your configuration file for all of the cloud providers: [Configuration File](https://github.com/Quansight/qhub/blob/master/docs/docs/aws/configuration.md) 
+## Usage
+### Installation pre-requisites
+* QHub is supported by macOS and Linux operating systems (Windows is **NOT** currently supported);
+* We recommend the adoption of virtual environments (`conda`, `pipenv` or `venv`) for successful usage. 
+
+### Install
+QHub's installation can be performed by using:
+* `conda`:
+  ```bash
+  conda install -c conda-forge qhub
+  ```
+  
+* or `pip` (instead):
+    ```bash
+    pip install qhub
+    ```  
+Once finished, you can check QHub's version (and additional CLI args) by typing:
+```bash
+qhub --help
+```
+If successful, the CLI output will be similar to the following:
 
-The configuration file is your user interface into deploying and scaling your data science environment. Each change triggers [Github Action] that will seamlessly update your infrastructure.
+![help_cli_output.png](docs/images/help_cli_output.png)
 
-![](docs/images/brand-diagram.png "architecture diagram")
+### Set up initialization
+#### 1. Cloud Provider
+The first required step is to **choose a Cloud Provider to host the project deployment**. The cloud installation is based
+on Kubernetes, but knowledge of Kubernetes is **NOT** required. By default, QHub uses DigitalOcean, AWS and GCP. If you 
+wish to use a different provider, make sure they support Kubernetes and Terraform and follow the suitable documentation
+to configure it.
```

Comment:
> If you wish to use a different provider, make sure they support Kubernetes and Terraform and follow the suitable documentation to configure it.

Not sure this statement adds value to the docs, and it's not quite true. I suggest removing it. 

QHub is currently supported by AWS, GCP, and DO.  Microsoft Azure is in the works from @balast, which might be fun to mention (#34). There's no estimate on when it will be ready

Reply:
> > If you wish to use a different provider, make sure they support Kubernetes and Terraform and follow the suitable documentation to configure it.
> 
> Not sure this statement adds value to the docs, and it's not quite true. I suggest removing it.

I am a bit confused now :/ Can you elaborate on this?
I asked this on Slack and heard that well, technically one is able to deploy anywhere where K8s and Terraform are supported. Happy to remove it, just trying to find the source of truth.

> QHub is currently supported by AWS, GCP, and DO. Microsoft Azure is in the works from @balast, which might be fun to mention (#34). There's no estimate on when it will be ready.

Roger that. I will add a mention to Azure, although I don't particularly love mentioning things that are not ready, and have no date to be. Will do to it, though.

Reply:
For the record: we had a chat about this and although **technically** 
> one is able to deploy anywhere where K8s and Terraform are supported

the learning curve is quite high, hence the suggestion to remove it from the docs.